### PR TITLE
Finish AddNewCryptoCurrency

### PR DESCRIPTION
**Pull request by Sebastian-Basmachi**

### CODE REVIEW

File: src/components/addCryptocurrencies/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      <Container center>
+        <Text labelInput>Add a Cryptocurrency</Text>
+        <Input
+          changeColor={inputText}
+          placeholder="Use a name or tiker symbol..."
+          value={inputText}
+          onChangeText={(text: string) => inputOnChange(text)}
+        />
+        <Container button>
+          <Button onPress={() => addCryptocurrency()}>
```
Comment (tomasdematos) [inline src/components/addCryptocurrencies/index.tsx]: use a callback:

 onPress={addCryptocurrency}

File: src/components/addCryptocurrencies/index.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      <TouchableOpacity onPress={() => backToInitialScreen()}>
+        <Text backToList>{backToList}</Text>
+      </TouchableOpacity>
+      <Container center>
+        <Text labelInput>Add a Cryptocurrency</Text>
+        <Input
+          changeColor={inputText}
+          placeholder="Use a name or tiker symbol..."
+          value={inputText}
+          onChangeText={(text: string) => inputOnChange(text)}
```
Comment (tomasdematos) [inline src/components/addCryptocurrencies/index.tsx]: use callback

onChangeText={inputOnChange}

File: src/app/actions/index.ts [hunk 3]
```diff
@@ -1,19 +1,49 @@
...
+    } catch (error) {
+      console.log(error);
+    }
+  };
+};
+
+export const postCryptocurrencyOfUser = (crypto: string) => {
+  return async function () {
+    try {
+      await AsyncStorage.setItem('cryptocurrenciesOfUser', crypto);
```
Comment (VicenBayugar) [inline src/app/actions/index.ts:26]: why not create a state for the cryptos that the user uses?

File: src/components/addCryptocurrencies/index.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    }
+  };
+
+  const backToInitialScreen = (): void => {
+    navigation.goBack();
+  };
+
+  return (
+    <>
+      <TouchableOpacity onPress={() => backToInitialScreen()}>
```
Comment (VicenBayugar) [inline src/components/addCryptocurrencies/index.tsx]: use onPress={backToInitialScreen}

File: src/components/cryptocurrenciesList/index.tsx [hunk 5]
```diff
@@ -36,13 +47,13 @@ const CryptocurrenciesList = () => {
         keyExtractor={item => item.id}
       />
       <Container>
-        <TouchableOpacity>
+        <TouchableOpacity onPress={() => addCryptocurrenciesScreen()}>
```
Comment (VicenBayugar) [inline src/components/cryptocurrenciesList/index.tsx]: onPress={addCryptoCurrenciesScreen}

File: src/app/actions/index.ts (hunk 3 above)
Comment (Sebastian-Basmachi) [reply src/app/actions/index.ts:26]: I create a state with redux that's call cryptocurrenciesOfUser, the action is getCryptocurrenciesOfUser

**Pull request merged**